quoridorcore/src/main/java/com/harush/zitoon/quoridor/core/model/WallData.java

```diff
@@ -59,7 +59,7 @@ public boolean isHorizontal() {
 		return isHorizontal;
 	}
 
-	public boolean isFirst() {
+	public boolean setIsFirst() {
```

Comment:
You meant to write `get` but wrote `set` instead. And anyway it should stay as `is` as I mentioned above regarding Java naming conventions.